test/QueryPlannerTestHelpers.h

```diff
@@ -283,6 +284,9 @@ constexpr auto OrderBy = [](const ::OrderBy::SortedVariables& sortedVariables,
 // Match a `UNION` operation.
 constexpr auto Union = MatchTypeAndOrderedChildren<::Union>;
 
+// Match a `SERVICE` operation.
+constexpr auto Service = MatchTypeAndOrderedChildren<::Service>;
```

Comment:
Please implement the matcher s.t. it also takes a matcher for the sibling tree and the `graphPatternAsString` 
etc.